scripts/configure-sentry-alerts.sh

```
@@ -0,0 +1,117 @@
+
+set -e
+
+# Configuration
+SENTRY_ORG="misty-step"
+SENTRY_PROJECT="scry"
+SENTRY_API_BASE="https://sentry.io/api/0"
+
+# Load API token from .env.local if not in environment
+if [ -z "$SENTRY_API_TOKEN" ]; then
+  if [ -f .env.local ]; then
+    export SENTRY_API_TOKEN=$(grep "^SENTRY_API_TOKEN=" .env.local | cut -d= -f2 | tr -d '"')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)